lib/index.js

```diff
@@ -29,6 +30,11 @@ class Client {
     throw "invalid URL argument"
   }
 
+  async put(actionType, data) {
+    const response = ActionApi.action(this.api, this.authToken, actionType, data)
+    return response
```

Comment:
Or  we can `return await response`

Reply:
I'd even shorten this code to `return ActionApi.action(this.api, this.authToken, actionType, data)`

Reply:
yes, sounds better.

Reply:
Done! now is:

`return ActionApi.action(this.api, this.authToken, actionType, data)`

Real test keep working, see the response below:

```
node index.js
test:  {
  help: 'http://localhost:5000/api/3/action/help_show?name=package_create',
  success: true,
  result: {
    license_title: null,
    maintainer: null,
    relationships_as_object: [],
    private: false,
    maintainer_email: null,
    num_tags: 0,
    id: 'd0cc5340-bfd4-47e6-a41c-94233d14a203',
    metadata_created: '2020-09-11T12:24:35.576884',
    metadata_modified: '2020-09-11T12:24:35.576889',
    author: null,
    author_email: null,
    state: 'active',
    version: null,
    creator_user_id: 'ceb0d8c2-352b-4a6c-b2c8-2a1a4190192b',
    type: 'dataset',
    resources: [],
    num_resources: 0,
    tags: [],
    groups: [],
    license_id: null,
    relationships_as_subject: [],
    organization: {
      description: 'my org',
      created: '2020-04-14T13:27:47.434967',
      title: 'myorg',
      name: 'myorg',
      is_organization: true,
      state: 'active',
      image_url: '',
      revision_id: 'bff67202-a21e-470b-9b67-14e65ad85fa2',
      type: 'organization',
      id: 'f3fce98a-4750-4d09-a5d1-d5e1b995a2e8',
      approval_status: 'approved'
    },
    name: 'my_dataset_5',
    isopen: false,
    url: null,
    notes: null,
    owner_org: 'f3fce98a-4750-4d09-a5d1-d5e1b995a2e8',
    extras: [],
    title: 'my_dataset_5',
    revision_id: '19b38438-4d26-46f8-ae8b-bef7e7e7cafc'
  }
}
```